students/junghoon/user/urls.py

```diff
@@ -1,6 +1,8 @@
 from django.urls import path
-from .views import SignupView
+
+from .views import SignupView, LoginView
 
 urlpatterns = [
     path('/signup', SignupView.as_view()),
+    path('', LoginView.as_view()),
```

Comment:
url은 아래와 같이 수정하여 8000/users/login으로 구현해 주시면 좋을 것 같습니다.
8000/users로 POST method를 실행하면 보통 user의 정보를 등록한다고 생각하는데 이는 회원 가입과 어울립니다.

```suggestion
    path('/login', LoginView.as_view()),
```

Reply:
아..확인했습니다.